GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -134,6 +114,28 @@ bool ExporterHelper::ExportProjectForPixiPreview(
     previousTime = LogTimeSpent("Events code export", previousTime);
   }
 
+  gd::Project exportedProject = options.project;
+
+  if (!options.fullLoadingScreen) {
+    // Most of the time, we skip the logo and minimum duration so that
+    // the preview start as soon as possible.
+    exportedProject.GetLoadingScreen().ShowGDevelopSplash(false);
+    exportedProject.GetLoadingScreen().SetMinDuration(0);
+  }
+
+  // Export resources (*before* generating events as some resources filenames
+  // may be updated)
+  ExportResources(fs, exportedProject, options.exportPath);
+
+  previousTime = LogTimeSpent("Resource export", previousTime);
+
+  // Compatibility with GD <= 5.0-beta56
+  // Stay compatible with text objects declaring their font as just a filename
+  // without a font resource - by manually adding these resources.
+  AddDeprecatedFontFilesToFontResources(
+      fs, exportedProject.GetResourcesManager(), options.exportPath);
+  // end of compatibility code
```

Comment:
Do you think it's ok to move this after the code generation?
I guess the generated code is using resource names so changing the resource pathes should probably not impact the code generation.

Reply:
Maybe... though the code generation could in theory raise an error because somehow you're using an action/condition with a "fontResource" and this resource does not exist.
I think the risk is small, but exists... What's the reason behind you wanting to move this?

Side note:
- At some point I experimented about exporting the resources in an asynchronous matter: we issue commands to copy files, which are launched on the Node.js side, and at the end of the export (or whenever the content of a file is needed, but this should not be required I think) we wait on all copies to be done.
  The idea is to parallelise the I/O (copying exports) and the events generation (CPU work, then I/O at the end).
  
  (This is a different problem though, so if we do this it should be done in another PR :) Don't make your work here more complex with this optimisation for now)

Reply:
It avoid to parse the expressions when exporting a second time which save around 10% of the exporting time. If it's risky, it can wait a different release.